README.md

```diff
@@ -392,3 +393,24 @@ For new snippets the general steps are
 ```
 
 [⬆ back to top](#table-of-contents)
+
+### Anagram Check
+
+```java
+public boolean isAnagram(String s1, String s2) {
+	var l1 = s1.length();
+	var l2 = s2.length();
+	int [] arr1 = new int[256];
+	int [] arr2 = new int[256];
```

Comment:
var arr1 = new int[256];

Reply:
$ jshell
|  Welcome to JShell -- Version 11.0.4
|  For an introduction type: /help intro

jshell> var arr1 = new int[256];
arr1 ==> int[256] { 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, ... , 0, 0, 0, 0, 0, 0, 0, 0 }

Reply:
Thanks for the information. Fixed 😄